pscheduler-archiver-rabbitmq/rabbitmq/archive

```diff
@@ -106,12 +106,24 @@ class AMQPExpiringConnection(object):
         self.expire_time = expire_time
 
         parsed_url = urllib.parse.urlparse(url)
-        if parsed_url.scheme not in ["amqp", "amqps"]:
+
+        # Set default port and SSL flag based on URL scheme
+        if (parsed_url.scheme == "amqp"):
+            port=5672
```

Comment:
Small nit:  Put spaces around the `=` in assignments so pylint doesn't complain.

Looks fine otherwise.

Reply:
Thank you for the review, and changes pushed. Please let me know if there's anything else, or if you would like it squashed.